resources/l10n/template.pot

```diff
@@ -6,9 +6,9 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: LOOT 0.10.3\n"
+"Project-Id-Version: LOOT 0.11.0\n"
 "Report-Msgid-Bugs-To: https://github.com/loot/loot/issues\n"
-"POT-Creation-Date: 2017-06-18 07:31+0100\n"
+"POT-Creation-Date: 2017-07-06 12:57+0200\n"
```

Comment:
Eh. I didn't intend to change/update `template.pot` too, but I guess `/scripts/git/pre-commit` wasn't run with or since 221d3ef3430caed562dff36dafbc4013fd7ae8a7, and I don't feel like making a new commit just to separate out this change. C'est la vie.